### ISSUE TITLE
Details about baseline resnet-101 in paper

### ISSUE THREAD

**Issue by zhifanlight** [Kevinz-code/CSRA#17]
Hi, thanks for your excellent work! But I'm confused of the detail about baseline-model settings in your paper.

Take training resnet-101 without cutmix on coco2014 as an example: 

With the following training configurations as baseline setting, I get 81.3 mAP after 7 epochs (30 in total, still in training process...), which is much higher than that in your paper (79.4 mAP). 
`python main.py --num_heads 4 --lam 0 --dataset coco --num_cls 80 --checkpoint coco14/resnet101`

So, what is the correct settings to reproduce the baseline result as in your paper? Thanks again.

**Comment by Kevinz-code** [comment]
Hi, thank you for reproducing our work and the baseline methods.

When we run baseline method on COCO, the learning rate and other configs follow previous multi-label work, e.g. [ML-GCN](https://openaccess.thecvf.com/content_CVPR_2019/papers/Chen_Multi-Label_Image_Recognition_With_Graph_Convolutional_Networks_CVPR_2019_paper.pdf), [MS-CMA](https://arxiv.org/pdf/1912.07872.pdf). Specifically, the learning rate for backbone and fc are 0.01 (also mentioned in our paper), step_size is 15, and we train about 40 epochs. 

When we use our method, since CSRA is a special module, we enlarge fc's learning rate to 0.1 for faster convergence, and step size and total epochs have been shortened consequently.

Best,
Authors.
